atlas/query_node_API_sketch.graphql

```diff
@@ -0,0 +1,196 @@
+
+enum Language {
+  Chinese
+  English
+  Arabic
+  Portugese
+  French
+  # ...
+}
+
+type Channel {
+
+  id: ID!
+
+  # Id of underlying entity.
+  entityID: BigInteger!
+
+  owner: Member!
+
+  handle: String!
+
+  description: String!
+
+  coverPhotoURL: String!
+
+  avatarPhotoURL: String!
+
+  isPublic: Bool!
+
+  isCurated: Boolean!
+
+  language: Language
+
+  videos: [Video!]
+}
+
+type Category {
+
+  id: ID!
+
+  # Id of underlying entity.
+  entityID: BigInteger!
+
+  name: String!
+
+  videos: [Video!]
```

Comment:
Same here, I guess those should be paginated

Reply:
no.